### PR TITLE
Security fix: add http headers `'X-Content-Type-Options':'nosniff'` and `'Referrer-Policy':'no-referrer'`

### PR DESCRIPTION
**Pull request by gadomsky**

### CODE REVIEW

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/util/ContentTypeOptionsSecureSupport.scala [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package pl.touk.nussknacker.ui.util
+
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.server.{Directive0, Route}
+
+object ContentTypeOptionsSecureSupport extends Directive0 {
+  val header = RawHeader("X-Content-Type-Options", "nosniff")
+
+  override def tapply(f: Unit => Route): Route = WithHeaders(header).tapply(f)
```
Comment (mproch) [inline ui/server/src/main/scala/pl/touk/nussknacker/ui/util/ContentTypeOptionsSecureSupport.scala:9]: why do we need it? Maybe just 
`
   apply(): Directive0 = WithHeaders(header)
`

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/util/WithDirectives.scala [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.model.{HttpMethods, HttpResponse, StatusCodes}
+import akka.http.scaladsl.server.Directive0
+import akka.http.scaladsl.server.Directives.{complete, respondWithHeaders}
+
+object WithDirectives {
+  def apply(directives: Directive0*): Directive0 = directives.reduce((d1, d2) => d1.and(d2))
+}
+
+object WithHeaders {
```
Comment (mproch) [inline ui/server/src/main/scala/pl/touk/nussknacker/ui/util/WithDirectives.scala:12]: Maybe WithHeadersIfNotOptions or sth?

Comment (gadomsky) [reply ui/server/src/main/scala/pl/touk/nussknacker/ui/util/WithDirectives.scala:12]: Why? For request method `OPTIONS` we return those headers

Comment (mproch) [reply ui/server/src/main/scala/pl/touk/nussknacker/ui/util/WithDirectives.scala:12]: sorry, I misread it :)

**Review by mproch — APPROVED** [review]

**Pull request merged**